src/utils/assertLocationUtils.tsx

```diff
@@ -41,9 +55,16 @@ export const assertLocationTxn = async (
   }
 
   const antennaGain = decimalGain * 10
-  const stakingFee = updatingLocation
-    ? Transaction.stakingFeeTxnAssertLocationV1
-    : 0
+  let stakingFee = 0
+  if (updatingLocation) {
+    if (dataOnly) {
```

Comment:
Does this look right @tyler-whitman ?

Reply:
Yeah looks good, just realized that `hasFreeLocationAssert` doesn't return a promise though so we can remove the `await`